SchoolExplorer.Api/Controllers/SchoolDistrictController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using SchoolExplorer.Api.Constants;
+using SchoolExplorer.Application.Common;
+using SchoolExplorer.Application.Dtos;
+using SchoolExplorer.Application.Services;
+
+namespace SchoolExplorer.Api.Controllers
+{
+	[Route($"{ApiRoutes.Base}/[{ApiRoutes.Controller}]")]
+	[ApiController]
+	public class SchoolDistrictController : ControllerBase
+	{
+		private readonly IValidator<CreateSchoolDistrictDto> _validator;
+		private readonly ISchoolDistrictService _schoolDistrictService;
+		public SchoolDistrictController(ISchoolDistrictService schoolDistrictService, IValidator<CreateSchoolDistrictDto> validator)
+		{
+			_validator=validator;
+			_schoolDistrictService=schoolDistrictService;
+		}
+		[HttpPost]
+		public async Task<IActionResult> CreateSchoolDistrictAsync([FromBody] CreateSchoolDistrictDto schoolDistrict)
+		{
+			try
+			{
+				var validatorResult = _validator.Validate(schoolDistrict);
+				if (!validatorResult.IsValid)
+				{
+					return BadRequest(ResponseMessages.InvalidData);
```

Comment:
This isn’t useful. There are good details in the validation message use the same.

Reply:
using validation message.